scripts/set_composites_names.py

```
@@ -0,0 +1,18 @@
+import argparse
+import requests
+import constant
+
+parser = argparse.ArgumentParser(description='Takes the name of the composites elements in directory server and apply it to them in netmod-server', )
+
+print("Getting composites lists from directory-server: " + constant.GET_DIRECTORY_ELEMENTS)
+directory_composites_response = requests.get(
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)